packages/extensions-sdk/src/cli/commands/create.ts

```diff
@@ -71,12 +71,13 @@ export default async function create(type: string, name: string, options: Create
 		keywords: ['directus', 'directus-extension', `directus-custom-${type}`],
 		[EXTENSION_PKG_KEY]: {
 			type,
-			path: 'dist/index.js',
+			path: `dist/${type}s/${name}/index.js`,
```

Comment:
I don't think this is accurate. The default `build` command renders to `dist/index.js`, not `dist/panels/my-panel/index.js` for example